gitclone/gitclone_test.go

```diff
@@ -0,0 +1,411 @@
+package gitclone
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/bitrise-io/bitrise-init/step"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/command/git"
+)
+
+const always = 9999
+const rawCmdError = "dummy_cmd_error"
+
+func Test_checkoutState(t *testing.T) {
+	tests := []struct {
+		name       string
+		cfg        Config
+		cmdOutputs map[string]commandOutput
+		want       *step.Error
```

Comment:
`wantErr` would more reflect that this is the expected error.

Reply:
Usually wantErr bool, but in this case makes sense, renamed.